src/Containers/FavPage/action.js

```diff
@@ -0,0 +1,16 @@
+export const ADD_FAV = 'ADD_FAV';
+export const WITHDRAW_FAV = 'WITHDRAW_FAV';
+
+export function addFav(params) {
+  return {
+    type: ADD_FAV,
+    params
```

Comment:
The related `data` of an action should be contained in a `payload` attribute.

Reply:
ok. I'll consider it next time.